src/main/java/org/spongepowered/api/util/SpongeApiTranslationHelper.java

```diff
@@ -22,4 +22,43 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-@org.spongepowered.api.util.annotation.NonnullByDefault package org.spongepowered.api.text.translation.locale;
+package org.spongepowered.api.util;
+
+import com.google.common.base.Function;
+import org.spongepowered.api.text.Text;
+import org.spongepowered.api.text.Texts;
+import org.spongepowered.api.text.translation.ResourceBundleTranslation;
+
+import java.util.Locale;
+import java.util.ResourceBundle;
+
+import javax.annotation.Nullable;
+
+/**
+ * This class provides translations for strings within SpongeAPI. Plugins should consult an implementaion of Translation for help.
+ */
+public class SpongeApiTranslationHelper {
```

Comment:
Maybe we should deprecate this with the reason for internal SpongeAPI usage only so developers will actually see they shouldn't use this.